docker-compose.yml

```diff
@@ -18,10 +21,17 @@ services:
     restart: always
     ports:
       - 80:8080
+    networks:
+      - stringer-network
     environment:
       - SECRET_KEY_BASE=<your configuration>
       - ENCRYPTION_PRIMARY_KEY<your configuration>
       - ENCRYPTION_DETERMINISTIC_KEY=<your configuration>
       - ENCRYPTION_KEY_DERIVATION_SALT=<your configuration>
       - PORT=8080
       - DATABASE_URL=postgres://db_user:super_secret_password@postgres:5432/stringer
+
+networks:
+  stringer-network:
+    external: false
+    name: stringer-network
```

Comment:
Can you help me to understand what this change does and why it's an improvement?

Reply:
it's ensure when you create your stringer from docker-compose file to create dedicated network to stringer, containing the postgres container and the web app. This prevent stringer web app to try to connect to another postgres container (for nextcloud or joplin for example). 

Like this you can just throw your docker compose in portainer stack and just run it. Independantly how many postgres container you have in your local docker It will work because their are on the same network

Reply:
Thanks for the explanation! This doesn't sound like it's going to cause any user facing impact. Is that right, or are they going to have to make some change on their end?

Reply:
No,it seems no. if new user follow documentation. It will create corresponding network and then if they drop the docker compose in some app like Portainer it will create the corresponding network too